animation_nodes/nodes/spline/get_partial_length.py

```diff
@@ -20,7 +36,23 @@ def drawAdvanced(self, layout):
         col.active = self.parameterType == "UNIFORM"
         col.prop(self, "resolution")
 
-    def execute(self, spline, start, end):
+    def getExecutionFunctionName(self):
+        if any([self.useSplineList, self.useStartList, self.useEndList]):
+            return "execute_MultipleSplines_MultipleStarts_MultipleEnds"
+        else:
+            return "execute_SingleSpline_SingleStart_SingleEnd"
+
+    def execute_MultipleSplines_MultipleStarts_MultipleEnds(self, spline, start, end):
+        splines = VirtualPyList.create(spline, None)
```

Comment:
The default value should be an empty spline, otherwise you get an error.

Reply:
Okay. So which one use here Poly or Bezier?

Reply:
@3DSinghVFX  It wouldn't matter in this case, since isEvaluable will be false. So just create a Poly one.

Reply:
Okay.